src/users/components/invite-user-permissions/invite-user-permission.component.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormGroup } from '@angular/forms';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
+import { of } from 'rxjs';
+import { InviteUserPermissionComponent } from './invite-user-permission.component';
+
+describe('Invite User Permission Component', () => {
+
+  let component: InviteUserPermissionComponent;
+  let fixture: ComponentFixture<InviteUserPermissionComponent>;
+
+  const mockFeatureToggleService = jasmine.createSpyObj('FeatureToggleService', ['getValue']);
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      schemas: [NO_ERRORS_SCHEMA],
+      declarations: [ InviteUserPermissionComponent ],
+      providers: [
+        {
+          provide: FeatureToggleService,
+          useValue: mockFeatureToggleService
+        }
+      ]
+    }).compileComponents();
+    mockFeatureToggleService.getValue.and.returnValue(of(true));
+    fixture = TestBed.createComponent(InviteUserPermissionComponent);
+    component = fixture.componentInstance;
+    component.inviteUserForm = new FormGroup({});
+    fixture.detectChanges();
+  });
+
+  it('should create component', () => {
```

Comment:
Is this really the only test? It's not a very meaningful one, especially since you've added the `NO_ERRORS_SCHEMA`. I think you need _something_ that tests the functionality of the component, not just that it doesn't blow up the minute you try to create it - e.g., test that the checkbox is there when `grantCaseAccessAdmin$` evaluates to try and test that it's not when it evaluates to false.

Reply:
thanks good shout, I just firstly tried to add the test file back, but now I added more tests for the logic I changed.